features/gamepad-extensions.yml

```diff
@@ -0,0 +1,23 @@
+name: gamepad
```

Comment:
The spec itself is called the "Gamepad Extensions", so that's why I elected to call it as such.  It is true that these updates are tailored towards XR controllers, but looking at some of these interfaces, I could see them used for a non-XR gamepad.  For example, the DualShock 4 and DualSense controllers have a gyroscope in them, which could be reflected through `GamepadPose.orientation`.  They also have a touchpad, which could be reflected by `GamepadTouch`.

Reply:
Got it. Though our overarching goal is to design features around what developers would want to use in their apps, not around how specs are structured. So if a spec contains one set of interfaces for doing X, and another set of interfaces for doing Y, we would create 2 separate features: X and Y.
I think this is likely due to how we generated a bunch of the draft features, which we did by using a script that looped over specs.

From what you said, it seems like there could almost be one feature for gamepad XR headset support, one for gamepad orientation, and one for gamepad touch support.
What do you think?

Also pinging @ddbeck on this one, to help make a decision re splitting this feature.